client/apollo/js/View/Track/AnnotTrack.js

```diff
@@ -2294,6 +2331,71 @@ define([
                     'class': "annotation_info_editor_button"
                 }, commentButtons);
 
+                var replacementDiv = dojo.create("div", {'class': "annotation_info_editor_section"}, content);
```

Comment:
you don't want to do this if feature.type.name is a 'gene' or a 'pseudogene' . . . such as, you might want to move most of this to another method if possible.